definitions/divorce/json/CaseEvent.json

```diff
@@ -2745,5 +2745,16 @@
     "PreConditionState(s)": "*",
     "PostConditionState": "*",
     "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2019",
+    "CaseTypeID": "DIVORCE",
+    "ID": "allowPetitionerToAmend",
+    "Name": "Allow Petitioner to amend",
+    "Description": "Allow Petitioner to amend",
+    "DisplayOrder": 23,
+    "PreConditionState(s)": "AwaitingHWFDecision;Submitted;Issued;Rejected;PendingRejection;solicitorAwaitingPaymentConfirmation;AosAwaiting;AosStarted;AosSubmittedAwaitingAnswer;AwaitingDecreeNisi;AosOverdue;AwaitingReissue;AosCompleted;AosAwaitingSol;AosPreSubmit;AosDrafted;AwaitingService",
```

Comment:
Happy this is now possible. In the past we would have had to create 17 separate events to do that :D

Reply:
So this is something that i took a bit of time to look into and didnt get much of an answer about. I see that we are doing this for `EnterAOSOffline_Respondent` as so i figured that this would be okay.
It is important that this does work as we will be doing many more events like this in the future.